web-ui/components/EditorPanel.vue

```diff
@@ -16,15 +16,12 @@
       <ExecuteButton @click="$emit('run')" />
       <div id="editor-panel-header-handle" @mousedown="startDrag" />
     </div>
-    <Editor
```

Comment:
Why have you gotten rid of the editor here? It seems to have broken the editor completely.